converter/har/converter.go

```diff
@@ -97,16 +102,20 @@ func Convert(h HAR, includeCodeCheck bool, returnOnFailedCheck bool, batchTime u
 		sort.Sort(EntryByStarted(entries))
 
 		if nobatch {
-			var recordedRedirectUrl string
+			var recordedRedirectURL string
+			var recordedRestID string
```

Comment:
It seems like you've removed the setting of `recordedRedirectURL` and `recordedRestID` with the latest commit, but the variables themselves and checks with them that will never pass are still in the code

Reply:
good catch. I meant to remove recordedRestID, but recordedRedirectURL I meant to not remove the use of :) 

fixed now!